docs/site/Troubleshooting.md

```diff
@@ -0,0 +1,93 @@
+---
+lang: en
+title: 'Troubleshooting'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Troubleshooting.html
+---
+
+In general, when you are developing an application, use the `npm` command to run
+it. This enables you to see stack traces and console output immediately.
+
+For example:
+
+```
+$ cd myapp
+$ npm start
+```
+
+LoopBack 4 also provides multiple ways to help you verify or diagnose the app.
+
+## Setting Debug Strings
+
+If you get an error but it doesn't provide enough information, you can turn on
+debug mode by starting your application using `DEBUG=<DEBUG_STRING> npm start`.
+[Setting debug strings](Setting-debug-strings.md) for details.
+
+## Running VS Code Tasks
+
+If you're using Visual Studio Code, refer to
+[Debugging with VS Code page](Debugging-with-vscode.md) for debugging tips.
+
+## Running Tests with Mocha
+
+You can debug tests by running mocha commands as documented on page
+[Debugging with Mocha tests](Debugging-tests-with-mocha.md).
```

Comment:
@bajtos Thank you for review, I am following the instructions in story https://github.com/strongloop/loopback-next/issues/5701, and the list contains all these tips from different angles. That's why my understanding is this page wants to cover all the troubleshooting related topics.

Reply:
I see, it's unfortunate that #5701 is open to different interpretations.

The problem I see in your current version: as a LB user troubleshooting a problem in my application, when I open the debugging guide, I see instructions that do not apply to my situation - the launch tasks created by `lb4 app` are different, there is no `packages` folder in my project, etc.

In my view, most of the documentation at loopback.io is for LoopBack users (application and extension developers). 

Content for contributors (to loopback-next and other repositories) belongs to "contrib" section of our documentation, which is currently spread between https://loopback.io/doc/en/lb4/code-contrib-lb4.html and https://loopback.io/doc/en/contrib/.

I feel it's super important to make it clear which content is for whom, to avoid confusion and frustration.

I am proposing the following way forward:

- In the new pages you are adding to `Troubleshooting` section of the main docs, provide information how to debug LoopBack applications and extensions.
- Information on debugging loopback-next should go to https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md. VSCode-specific information belongs to https://github.com/strongloop/loopback-next/blob/master/docs/site/Developing-with-vscode.md, see also https://github.com/strongloop/loopback-next/blob/master/docs/site/Developing-with-webstorm.md
- If you like, you can cross-link these different resources.

Reply:
👍 @bajtos I post my plan in https://github.com/strongloop/loopback-next/pull/6371#issuecomment-696811055 ptal, thanks